packages/phishing-controller/src/PhishingController.ts

```diff
@@ -861,6 +909,154 @@ export class PhishingController extends BaseController<
     return combinedResponse;
   };
 
+  /**
+   * Map chain ID to chain name for the API.
+   *
+   * @param chainId - The chain ID.
+   * @returns The chain name.
+   */
+  readonly #chainIdToName: Record<string, string> = {
+    '0x1': 'ethereum',
+    '0x89': 'polygon',
+    '0x38': 'bsc',
+    '0xa4b1': 'arbitrum',
+    '0xa86a': 'avalanche',
+    '0x2105': 'base',
+    '0xa': 'optimism',
+    '0x76adf1': 'zora',
+    '0xe708': 'linea',
+    '0x27bc86aa': 'degen',
+    '0x144': 'zksync',
+    '0x82750': 'scroll',
+    '0x13e31': 'blast',
+    '0x74c': 'soneium',
+    '0x79a': 'soneium-minato',
+    '0x14a34': 'base-sepolia',
+    '0xab5': 'abstract',
+    '0x849ea': 'zero-network',
+    '0x138de': 'berachain',
+    '0x82': 'unichain',
+    '0x7e4': 'ronin',
+    '0x127': 'hedera',
+  };
```

Comment:
For better type safing we could use `as const` here. You can find more details about the [const assertion](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions), as well as how it improves type inferrence in our [contributor docs](https://github.com/MetaMask/contributor-docs/blob/main/docs/typescript.md#type-inference)